src/pcapi/utils/includes.py

```diff
@@ -51,19 +51,31 @@
             "remainingQuantity",
         ],
     },
-    {
-        "key": "venue",
-        "includes": [
-            {"key": "managingOfferer", "includes": ["-validationToken", "isValidated", "bic", "iban"]},
-            "-validationToken",
-            "isValidated",
-            "bic",
-            "iban",
-        ],
-    },
 ]
 
+venue_key_with_bank_informations = {
+    "key": "venue",
+    "includes": [
+        {"key": "managingOfferer", "includes": ["-validationToken", "isValidated", "bic", "iban"]},
+        "-validationToken",
+        "isValidated",
+        "bic",
+        "iban",
+    ],
+}
+
+venue_key_without_bank_informations = {
+    "key": "venue",
+    "includes": [
+        {"key": "managingOfferer", "includes": ["-validationToken", "isValidated"]},
+        "-validationToken",
```

Comment:
A priori, on exporte aussi `Venue.bookingEmail` pour la webapp qui n'en a sûrement pas plus besoin que l'IBAN ou le BIC. On pourrait donc aussi l'exclure.


```suggestion
        "-bookingEmail",
        "-validationToken",
```